src/sourcemaps/utils/other-wizards.ts

```diff
@@ -146,3 +159,24 @@ https://docs.sentry.io/platforms/javascript/guides/${frameworkSlug}/sourcemaps/t
       return undefined;
   }
 }
+
+function runReactNativeWizard(): Promise<void> {
+  const [runner, ...wizardArgs] = [...process.argv];
+  wizardArgs.push('--integration', 'reactNative');
+
+  try {
+    childProcess.spawnSync(runner, wizardArgs, {
+      cwd: process.cwd(),
+      stdio: 'inherit',
+    });
+  } catch {
+    clack.log.error(
+      `Could not redirect to the react-native wizard.\nPlease re-run the wizard manually with the \`${chalk.cyan(
+        '-i reactNative',
+      )}\` argument`,
+    );
+    return Promise.reject();
```

Comment:
In the case the react wizard fails, will this message be printed? If so, I don't think that is fully fitting.

Reply:
Hmm it's a good point. Either the invocation fails (in which case the message would make sense IMO) or the RN wizard fails later (in which the message doesn't make sense) but we can't distinguish between the cases here. Let's remove the log then. In case the wizard fails, users should get another error anyway.